<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmNewProducto</class>
 <widget class="QDialog" name="FrmNewProducto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>32</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>nombre :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDescripcion">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>58</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripci√≥n :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio :</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SbPrecio">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sBStock">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Stock :</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dEFecha">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha Vecimiento : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdGrabar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CmdCerrar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
